.claude/commands/pr-review.md

```
@@ -0,0 +1,160 @@
+This command facilitates a collaborative PR review process where:
+1. Claude shows the diff of the current branch vs main
+2. User raises issues they've found
+3. Claude corrects each issue
+4. Issues of severity Medium, Major, or Critical are added to the appropriate errors-to-avoid documentation
+
+## Execution Flow
+
+### Step 1: Show PR Diff
+
+Run the following to get the diff:
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)